llvm/test/MC/Xtensa/Options/code_density-invalid.s

```diff
@@ -0,0 +1,17 @@
+# RUN: not llvm-mc -triple xtensa --mattr=+density %s 2>&1 | FileCheck %s
+
+LBL0:
+
+# Out of range immediates
+
+# imm1n_15
+addi.n a2, a3, 20
```

Comment:
Would you please add a test case for imm=0?

Reply:
I added test case.